src/components/Superfest/QuestCard/QuestCard.tsx

```diff
@@ -110,17 +115,34 @@ export const QuestCard = ({
           </FlexCenterRowBox>
           {points ? (
             <FlexCenterRowBox>
+              {apy > 0 ?
               <XPDisplayBox
                 active={active}
-                bgcolor={!completed ? '#ff0420' : '#42B852'}
+                bgcolor={'#ff0420'}
               >
                 <SoraTypography
                   fontSize="14px"
                   fontWeight={700}
                   lineHeight="18px"
                   color={'#ffffff'}
                 >
-                  {`+${points}`}
+                  {`${Number(apy).toFixed(1)}%`}
+                </SoraTypography>
+                <XPIconBox marginLeft="4px">
+                    <APYIcon size={20} />
+                </XPIconBox>
+              </XPDisplayBox> : undefined}
+              <XPDisplayBox
+                active={active}
+                bgcolor={!completed ? '#31007A' : '#42B852'}
```

Comment:
Would be good to use the palette

Reply:
So these colors are not part of the palette as they are one-off colors for a "check" and specific icon. I can include it into the palette but I fear we forget it.